includes/render.php

```diff
@@ -56,54 +60,71 @@ class Mad_Mimi_Form_Fields {
 	private static $cycle = 0;
 
 	public function dispatch_field( $field, $cycle = 1 ) {
-		if ( ! is_object( $field ) || ! method_exists( __CLASS__, $field->type ) )
+
+		if ( ! is_object( $field ) || ! method_exists( __CLASS__, $field->type ) ) {
 			return;
+		}
 
 		self::$cycle = absint( $cycle );
 
 		call_user_func( array( __CLASS__, $field->type ), $field );
+
 	}
 
 	public function get_form_id( $field_name ) {
+
 		// since HTML ID's can't exist in the same exact spelling more than once... make it special.
 		return esc_attr( sprintf( 'form_%s_%s', self::$cycle, $field_name ) );
```

Comment:
I know you've not changed this line... but this escape call is redundant really - it should be done when its output.